examples/micronaut/README.md

```diff
@@ -20,7 +20,7 @@ docker run -d -p 8080:8080 micronaut-jib:0.1
 ```
 ```shell
 curl localhost:8080/hello
-> Hello World
+> Hello Jib
```

Comment:
Maybe we should change the code instead, the gif still says Hello World.

Reply:
Ah, good point, I'll just change back to Hello World